app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -37,6 +44,21 @@ export default class WebhookEvents extends Component {
     } finally {
       this.loading = false;
     }
+    this.eventIds = this.filteredEventIds();
+    this.count = this.eventIds.length;
+    if (this.count === 0) {
+      this.redeliverEnabled = false;
+    }
+  }
+
+  filteredEventIds() {
+    return this.events.content
+      .filter(this.failedEvents)
+      .map((event) => event.id);
+  }
+
+  failedEvents(event) {
+    return event.status < 200 || (event.status > 299 && event.status !== 0);
```

Comment:
I think the parenthese is in the wrong spot, I think it is

 `return (event.status < 200 || event.status > 299) && event.status !== 0;`